index.html

```diff
@@ -0,0 +1,177 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,900&display=swap');
+        </style>
+
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css" integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc" crossorigin="anonymous">
+
+<link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.14.0/css/all.css"
+      integrity="sha384-HzLeBuhoNPvSl5KYnjx0BT+WB0QEEqLprO+NBkkk5gbc67FTaL7XIGa2w1L0Xbgc"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap');
+        </style>
+    <title>Visit Rwanda Festivals and Events</title>
+</head>
+<body>
+    <div class="social_icon">
+        <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+        </ul>
+    </div>
+    <header id="header">
+        <div class="logo">
+          <a href="index.html">
+          <img src="images/gorilla-logo.png" alt=""></a>
+        </div>
+        <nav id="nav-bar" class="nav-bar d-none">
+          
+          <div class="nav-wrap">
+            <a class="nav-link" id="about" href="aboutpage.html" >About</a>
+            <a class="nav-link visited" href="#programs" >Program</a>
+            <a class="nav-link" href="#contents" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#sponsor" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+          
+        </div>
+        <div id="close-icon" class="close-icon d-none">
+            <i class="fas fa-times-circle"></i>
+          </div>
+      </header>
+    <div class="banner" >
+        <div class="background">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals <br> and Events</h2>
+        <p>If you’re looking to get out of the big city and back into nature, the annual Kwita Izina gorilla-naming ceremony is a special time to visit Volcanoes National Park. The annual naming ceremony takes place in Kinigi,</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+    </div>
```

Comment:
- [ ] The typography(font family, font-weight, font size,...), headings texts, paragraphs, and spacing contents of your _hero_/ _banner_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Once again, we actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** matches that of the provided template design

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/133246874-63c99980-f07e-4971-bee0-14ab0278df26.png)


**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)